proto/binlogdata.proto

```diff
@@ -497,6 +497,8 @@ message MinimalSchema {
 message VStreamOptions {
   repeated string internal_tables = 1;
   map<string, string> config_overrides = 2;
+  // Skip copy phase for these tables.
+  repeated string tables_to_skip_copy = 3;
```

Comment:
Let's reverse this: use `tables_to_copy` since that will _usually_ be the smaller number. For example if I am copying all the tables (say 20) and a new table has come, we will then just have one table to specify to be copied, rather than 20 to be skipped.

Reply:
done.